src/components/ui/loader/index.tsx

```diff
@@ -1,27 +1,40 @@
 import classnames from 'classnames';
 import { Component } from 'react';
 import injectStyle, { WithStylesProps } from 'react-jss';
-import ReactLoader from 'react-loader';
-import { Theme } from '../../../themes';
+import { Oval } from 'react-loader-spinner';
+import { inject } from 'mobx-react';
+import { FerdiumStores } from '../../../@types/stores.types';
 
-const styles = (theme: Theme) => ({
+const styles = () => ({
   container: {
     position: 'relative',
-    height: 60,
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    height: 'inherit',
   },
-  loader: {},
-  color: theme.colorText,
 });
 
 interface IProps extends WithStylesProps<typeof styles> {
   className?: string;
   color?: string;
+  size?: number;
+  loaded?: boolean;
+  stores?: FerdiumStores;
 }
 
+@inject('stores')
 class LoaderComponent extends Component<IProps> {
   render() {
-    const { classes, className, color } = this.props;
-
+    const {
+      classes,
+      className,
+      size = 36,
+      color = '#FFF',
```

Comment:
Can we maybe set the default to be 'primary' -> settings.app.accentColor ? That way we can remove all `<Loader color='primary' />` to `<Loader />`. What do you think?

Reply:
Well, we have to either pass white in to certain loaders if white ain't the default prop, or pass primary if white is the default, up to you!

Reply:
Changed default color to be accentColor and now passing white color in as prop when needed: [`b4d3dd6` (#1410)](https://github.com/ferdium/ferdium-app/pull/1410/commits/b4d3dd6cb6ca30ff6db8a0a866339e1ba9d95683)